### PR TITLE
LL-3504 select fees strategy

### PR DESCRIPTION
**Pull request by LFBarreto**
- [LL-3594] wip: fee selection for eth
- [LL-3504] add select fee strat for btc
- [LL-3504] add translate label
- [LL-3504] clean code + add trans
- [LL-3504] change advanced mode to null
- bump live-common
- (SendAmountFields): fix onChange -> updateTransaction usage
It's the rework of the UI system for fees

### Type

UI Polish / Feature
### Context

[LL-3504]
### Parts of the app affected / Test plan

Send for BTC / ETH

[LL-3594]: https://ledgerhq.atlassian.net/browse/LL-3594
[LL-3504]: https://ledgerhq.atlassian.net/browse/LL-3504
[LL-3504]: https://ledgerhq.atlassian.net/browse/LL-3504
[LL-3504]: https://ledgerhq.atlassian.net/browse/LL-3504
[LL-3504]: https://ledgerhq.atlassian.net/browse/LL-3504
[LL-3504]: https://ledgerhq.atlassian.net/browse/LL-3504

### CODE REVIEW

**Comment by LFBarreto** [comment]
copy with review of #3738

**Review by machard — APPROVED** [review]

**Comment by JunichiSugiura** [comment]
Is this a right ticket?

> LL-3594 wip: fee selection for eth

File: src/renderer/components/SendFeeMode.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          onClick={toggleAdvanceMode}
+        >
+          <Trans i18nKey="send.steps.amount.advanced" />
+        </AdvancedText>
+      </ModeBox>
+    </Box>
+  );
+};
+
+export default withTranslation()(SendFeeMode);
```
Comment (JunichiSugiura) [inline src/renderer/components/SendFeeMode.js:86]: Could you avoid HOC when you can? useTranslate instead.

Comment (JunichiSugiura) [reply src/renderer/components/SendFeeMode.js:86]: But you don't even need it this time because Trans is used here.

**Review by JunichiSugiura — CHANGES_REQUESTED** [review]

**Comment by gre** [comment]
btw it's already part of live-common bump, so if we have to do changes, let's not forget to backreport them, but only if the changes are important to not invalidate ongoing QA.

**Comment by gre** [comment]
included in #3773

**Pull request closed without merging**